contracts/test/DelegateE2E.sol

```diff
@@ -85,7 +86,7 @@ contract TestUnitJBXBuybackDelegate is Test {
     jbSplitsStore = jbController.splitsStore();
 
     pool = IUniswapV3Pool(IUniswapV3Factory(0x1F98431c8aD98523631AE4a59f267346ea31F984).createPool(address(weth), address(jbx), 100));
-    pool.initialize(79228162514264337593543950336000000000); // 1 eth <=> 69420 jbx
+    pool.initialize(sqrtPriceX96); // 1 eth <=> 69420 jbx
```

Comment:
just a minor thing I see the current price is 1.75 mil jbx per eth https://info.uniswap.org/#/pools/0x48598ff1cee7b4d31f8f9050c2bbae98e17e6b17  should we use that instead ?